crates/subspace-node/src/domain/cli.rs

```diff
@@ -214,9 +218,13 @@ impl CliConfiguration<Self> for DomainCli {
         self.run.chain_id(is_dev)
     }
 
-    fn role(&self, is_dev: bool) -> Result<sc_service::Role> {
-        // is authority when operator is enabled or in dev mode
-        let is_authority = self.operator || self.run.validator || is_dev;
+    fn role(&self, _is_dev: bool) -> Result<sc_service::Role> {
+        if self.run.validator {
+            log::warn!("use `--operator-id` argument to run as operator")
```

Comment:
What about returning an error instead? That way no one will forget to specify operator ID. Otherwise we're keeping a footgun that some will definitely hit.